node/src/chain_spec.rs

```diff
@@ -365,7 +370,7 @@ pub fn pendulum_config() -> PendulumChainSpec {
 pub fn development_config() -> DevelopmentChainSpec {
 	// Give your base currency a unit name and decimal places
 	let mut properties = sc_chain_spec::Properties::new();
-	properties.insert("tokenSymbol".into(), "UNIT".into());
+	properties.insert("ForeignCurrencyId".into(), "UNIT".into());
```

Comment:
I think this actually has to remain as "tokenSymbol" everywhere. I don't think it is related to our `CurrencyId` enum.